### PR TITLE
Obj detect v003

### PR DESCRIPTION
**Pull request by dxaen**
Adds a new class to the list of classes we detect. But we adjust the labels of the classes we send to the server to keep the server implementation consistent.

### CODE REVIEW

File: CardScan/Classes/DetectedSSDBox.swift [hunk 1]
```diff
@@ -19,19 +19,24 @@ struct DetectedSSDBox {
         let XMax_ = XMax * Double(imageSize.width)
         let YMin_ = YMin * Double(imageSize.height)
         let YMax_ = YMax * Double(imageSize.height)
-
+
```
Comment (jaimejiyepark) [inline CardScan/Classes/DetectedSSDBox.swift]: I feel like @kingst is going to nit these spaces

**Comment by jaimejiyepark** [comment]
lgtm

File: CardScan/Classes/DetectedSSDBox.swift (hunk 1 above)
Comment (dxaen) [reply CardScan/Classes/DetectedSSDBox.swift]: Ok. Let me fix this real quick.

Comment (kingst) [reply CardScan/Classes/DetectedSSDBox.swift]: nit: let's remove this whitespace change

Comment (dxaen) [reply CardScan/Classes/DetectedSSDBox.swift]: Sure. Also, I tested it locally and the labels were correct.

**Pull request merged**